### DISCUSSION TITLE
Adding Temperature Forcing Instruments to Pymeasure

### DISCUSSION THREAD

**Discussion by jcarbelbide** [pymeasure/pymeasure#558] [Ideas]
I tried looking for temperature forcing instruments (ovens, thermostreams, etc.) but did not see any for pymeasure. I also did not see any discussions related to temperature forcing instruments. Are these out of the scope of pymeasure, or is this something that can be added? I have a some temperature forcing instruments at work for which I have written some rudimentary Python drivers that I could contribute to pymeasure if acceptable.

**Comment by LongnoseRob** [comment]
Hello @jcarbelbide,

I see no reason why thermal equiptment should not be included, there are already many motor-controllers implemented, so why should we not also feature temperature-control gear in different variations.

**Reply by jcarbelbide** [reply]
Great, thank you! I'll work on adapting some of the drivers I'm using for pymeasure.

**Reply by LongnoseRob** [reply]
Looking trough the insruments already implemented, I actually found one temperature control device: [Fluke 7341](pymeasure/instruments/fluke/fluke7341.py)

**Reply by jcarbelbide** [reply]
Awesome, I did not see that one. Thanks! I will reference it for the instruments I will work on. 

For some reason, your link didn't work for me, but here is the link for anyone else reading in case mine works: [Fluke 7341](https://github.com/pymeasure/pymeasure/blob/master/pymeasure/instruments/fluke/fluke7341.py)
